shared/constants/swaps.ts

```diff
@@ -166,7 +166,7 @@ const SWAPS_TESTNET_CHAIN_ID = '0x539';
 
 export const SWAPS_API_V2_BASE_URL = 'https://swap.metaswap.codefi.network';
 export const SWAPS_DEV_API_V2_BASE_URL = 'https://swap.dev-api.cx.metamask.io';
-export const GAS_API_BASE_URL = 'https://gas.api.cx.metamask.io';
+export const GAS_API_BASE_URL = 'https://gas.api.infura.io';
```

Comment:
If we're changing the URL, do we also need to update some swaps requests somewhere to include the Infura API key?

Reply:
Good call Matthew, apparently only gas prices used in the client code so fixed only in that function https://github.com/MetaMask/metamask-extension/pull/23717/commits/afbf07906dc84d6e5975912d2ce3b58a3c6ef0a0